app/api/v2/handlers/agent_api.py

```diff
@@ -33,9 +33,21 @@ async def get_agents(self, request: web.Request):
         agents = await self.get_all_objects(request)
         return web.json_response(agents)
 
-    @aiohttp_apispec.docs(tags=['agents'])
+    @aiohttp_apispec.docs(tags=['agents'],
+                          summary="Retrieve Agent Information",
```

Comment:
Just to make sure this is different from the `get_agents` description, can you change this summary to "Retrieve Agent by paw"?